person.rb

```diff
@@ -1,19 +1,19 @@
-require './nameable'
-require './capitalize_decorator'
+require_relative './nameable'
 
 # Class Person
 class Person < Nameable
-  def initialize(age, name: 'Unkown', parent_permission: true)
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
+  def initialize(age, name = 'Unkown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] Please add a method that creates rentals from the person and book class. The method should pass in the Person or the Book itself as a parameter in creating the rental. For example in the Person class, you can use this;
```ruby
def add_rental(date, book)
  Rental.new(date, self, book)
end
```